crates/cubecl-linalg/src/matmul/components/global/base.rs

```diff
@@ -70,7 +69,7 @@ pub trait Matmul<MS: MatmulSpec>: 'static + Send + Sync + MatmulKernel<Config: C
 
     /// Initialize the unloader at row m and column n
     fn init_unloader(
-        out: TensorOutput<MS::EG, MS::Args>,
+        out: VirtualTensor<MS::EG, ReadWrite>,
         m_offset: u32,
         n_offset: u32,
         batch_offset: u32,
```

Comment:
I really enjoy that the IO capacity is explicit. In comparison to the previous version, I immediately that `out` is mutable